test/Test/Toml/Gen.hs

```diff
@@ -56,11 +58,19 @@ genAnyValue = do
   let randI = toInteger <$> Gen.int (Range.constantBounded @Int)
   let randD = Gen.double $ Range.constant @Double (-1000000.0) 1000000.0
   let randT = Gen.text (Range.constant 0 256) Gen.alphaNum
```

Comment:
Also, let's perform some renaming. It's more conventional to name generators as `genSomething`.